pkg/component/gardeneradmissioncontroller/admissioncontroller.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Nit: In `gardenerapiserver` package, the corresponding file is called `gardener_apiserver.go`. Rename to `gardener_admission_controller.go`?